appengine/endpoints/README.md

```diff
@@ -1,7 +1,9 @@
 # Google Cloud Endpoints & App Engine Flexible Environment & Ruby
 
 This sample demonstrates how to use Google Cloud Endpoints on
-Google App Engine flexible environment using Ruby.
+Google App Engine flexible environment using Ruby. This sample requires that you
+have [Ruby](https://www.ruby-lang.org/en/documentation/installation/) 2.0.0 or
```

Comment:
Out of curiosity, did you run into any problems running this sample with previous versions of Ruby?

The stable, maintained versions of Ruby include 2.3* 2.2* and 2.1*.  Ruby 2.0* reached end of life [earlier this year](https://www.ruby-lang.org/en/news/2016/02/24/support-plan-of-ruby-2-0-0-and-2-1/).

This comment is 👍  but I'll come back through and update all READMEs to mention that samples are only intended to run on 2.1+ (and update travis to run 2.3, 2.2, 2.1)

Mostly curious if this blew up for you! 😄  Lemme know